config/routes.rb

```diff
@@ -30,7 +30,7 @@
              controller: "devise/two_step_verification_session"
     resource :two_step_verification,
              only: %i[show update],
-             path: "/users/two_step_verification",
+             path: "/account/two_step_verification",
```

Comment:
It took me a while to realise that this is OK because you can only set-up 2SV for your own account (i.e. admins can't configure it for other users' accounts). It might be worth stating that in the commit note, but I'll leave it to you to decide whether it's useful or not.